src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/csvjsonconvertor.py

```diff
@@ -0,0 +1,200 @@
+import csv
+import getopt
+import json
+import os
+import os.path
+import serialobj
+import sys
+sys.path.append(os.getenv('HELIX_SCRIPT_ROOT'))
```

Comment:
Is the “PYTHONPATH” environment variable set? It should be set by our code.

From: Deepak Shankargouda [mailto:notifications@github.com]
Sent: Tuesday, February 02, 2016 5:26 PM
To: dotnet/buildtools <buildtools@noreply.github.com>
Cc: Chad Nedzlek <chadnedz@microsoft.com>
Subject: Re: [buildtools] Python script to convert csv results to BV jsons (#426)


In src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/csvjsonconvertor.py<https://github.com/dotnet/buildtools/pull/426#discussion_r51666175>:

> @@ -0,0 +1,200 @@

> +import csv

> +import getopt

> +import json

> +import os

> +import os.path

> +import serialobj

> +import sys

> +sys.path.append(os.getenv('HELIX_SCRIPT_ROOT'))

I tried taking this off and running the script on an end machine with helix, it doesn't seem to recognize "helix"

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/buildtools/pull/426/files#r51666175>.